### ISSUE TITLE
npm run build  error for 4.0.0

### ISSUE THREAD

**Issue by jayzch** [vuejs/vue-devtools#497]
➜  vue-devtools git:(master) ✗ npm run build

> vue-devtools@3.1.7 build /Users/hulk/tmp/vue-devtools
> cd shells/chrome && cross-env NODE_ENV=production webpack --progress --hide-modules

Hash: 30a76ede939f03ac4fb8
Version: webpack 3.10.0
Time: 6726ms
                 Asset       Size  Chunks                    Chunk Names
           devtools.js     397 kB       0  [emitted]  [big]  devtools
            backend.js    20.9 kB       1  [emitted]         backend
               hook.js    1.87 kB       2  [emitted]         hook
           detector.js    2.62 kB       3  [emitted]         detector
              proxy.js  876 bytes       4  [emitted]         proxy
devtools-background.js    1.82 kB       5  [emitted]         devtools-background
         background.js       2 kB       6  [emitted]         background

ERROR in /Users/hulk/tmp/vue-devtools/node_modules/buble-loader?{"target":null,"objectAssign":"Object.assign"}!/Users/hulk/tmp/vue-devtools/node_modules/vue-loader/lib/selector.js?type=script&index=0!/Users/hulk/tmp/vue-devtools/src/devtools/components/SplitPane.vue
Module parse failed: Unexpected token (32:6)
You may need an appropriate loader to handle this file type.
|     {leftStyles: function leftStyles () {
|       return ( obj = {}
|       var obj;, obj[this.view === 'vertical' ? 'width' : 'height'] = ((this.boundSplit) + "%"), obj )
|     },
|     rightStyles: function rightStyles () {
 @ /Users/hulk/tmp/vue-devtools/src/devtools/components/SplitPane.vue 6:0-165 7:0-178
 @ /Users/hulk/tmp/vue-devtools/node_modules/buble-loader?{"target":null,"objectAssign":"Object.assign"}!/Users/hulk/tmp/vue-devtools/node_modules/vue-loader/lib/selector.js?type=script&index=0!/Users/hulk/tmp/vue-devtools/src/devtools/views/components/ComponentsTab.vue
 @ /Users/hulk/tmp/vue-devtools/src/devtools/views/components/ComponentsTab.vue
 @ /Users/hulk/tmp/vue-devtools/node_modules/buble-loader?{"target":null,"objectAssign":"Object.assign"}!/Users/hulk/tmp/vue-devtools/node_modules/vue-loader/lib/selector.js?type=script&index=0!/Users/hulk/tmp/vue-devtools/src/devtools/App.vue
 @ /Users/hulk/tmp/vue-devtools/src/devtools/App.vue
 @ /Users/hulk/tmp/vue-devtools/src/devtools/index.js
 @ ./src/devtools.js
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vue-devtools@3.1.7 build: `cd shells/chrome && cross-env NODE_ENV=production webpack --progress --hide-modules`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the vue-devtools@3.1.7 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/hulk/.npm/_logs/2018-01-13T02_41_05_372Z-debug.log

**Comment by yoyo837** [comment]
same error.

`ERROR in D:/source-control/git/vue-devtools/node_modules/buble-loader?{"target":null,"objectAssign":"Object.assign"}!D:/source-control/git/vue-devtools/node_modules/vue-loader/lib/selector.js?type=script&index=0!D:/source-control/git/vue-devtools/src/devtools/components/SplitPane.vue
Module parse failed: Unexpected token (32:6)
You may need an appropriate loader to handle this file type.
|     {leftStyles: function leftStyles () {
|       return ( obj = {}
|       var obj;, obj[this.view === 'vertical' ? 'width' : 'height'] = ((this.boundSplit) + "%"), obj )
|     },
|     rightStyles: function rightStyles () {
 @ D:/source-control/git/vue-devtools/src/devtools/components/SplitPane.vue 6:0-165 7:0-178
 @ D:/source-control/git/vue-devtools/node_modules/buble-loader?{"target":null,"objectAssign":"Object.assign"}!D:/source-control/git/vue-devtools/node_modules/vue-loader/lib/selector.js?type=script&index=0!D:/source-control/git/vue-devtools/src/devtools/views/components/ComponentsTab.vue
 @ D:/source-control/git/vue-devtools/src/devtools/views/components/ComponentsTab.vue
 @ D:/source-control/git/vue-devtools/node_modules/buble-loader?{"target":null,"objectAssign":"Object.assign"}!D:/source-control/git/vue-devtools/node_modules/vue-loader/lib/selector.js?type=script&index=0!D:/source-control/git/vue-devtools/src/devtools/App.vue
 @ D:/source-control/git/vue-devtools/src/devtools/App.vue
 @ D:/source-control/git/vue-devtools/src/devtools/index.js
 @ ./src/devtools.js`

`0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'build' ]
2 info using npm@5.6.0
3 info using node@v8.9.4
4 verbose run-script [ 'prebuild', 'build', 'postbuild' ]
5 info lifecycle vue-devtools@3.1.7~prebuild: vue-devtools@3.1.7
6 info lifecycle vue-devtools@3.1.7~build: vue-devtools@3.1.7
7 verbose lifecycle vue-devtools@3.1.7~build: unsafe-perm in lifecycle true
8 verbose lifecycle vue-devtools@3.1.7~build: PATH: C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;D:\source-control\git\vue-devtools\node_modules\.bin;C:\Python27\;C:\Python27\Scripts;C:\Program Files\Java\jdk1.8.0_144\bin;D:\apache-maven-3.5.0\bin;D:\apache-tomcat-8.0.44\bin;C:\Program Files\TortoiseSVN\bin;C:\ProgramData\Oracle\Java\javapath;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0\;C:\Program Files (x86)\Windows Kits\8.1\Windows Performance Toolkit\;C:\Program Files\TortoiseGit\bin;C:\Program Files\Git\cmd;C:\Program Files\nodejs\;C:\Users\yoyo8\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Microsoft VS Code\bin;C:\Users\yoyo8\AppData\Roaming\npm
9 verbose lifecycle vue-devtools@3.1.7~build: CWD: D:\source-control\git\vue-devtools
10 silly lifecycle vue-devtools@3.1.7~build: Args: [ '/d /s /c',
10 silly lifecycle   'cd shells/chrome && cross-env NODE_ENV=production webpack --progress --hide-modules' ]
11 silly lifecycle vue-devtools@3.1.7~build: Returned: code: 2  signal: null
12 info lifecycle vue-devtools@3.1.7~build: Failed to exec build script
13 verbose stack Error: vue-devtools@3.1.7 build: `cd shells/chrome && cross-env NODE_ENV=production webpack --progress --hide-modules`
13 verbose stack Exit status 2
13 verbose stack     at EventEmitter.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\index.js:285:16)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at EventEmitter.emit (events.js:214:7)
13 verbose stack     at ChildProcess.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at emitTwo (events.js:126:13)
13 verbose stack     at ChildProcess.emit (events.js:214:7)
13 verbose stack     at maybeClose (internal/child_process.js:925:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
14 verbose pkgid vue-devtools@3.1.7
15 verbose cwd D:\source-control\git\vue-devtools
16 verbose Windows_NT 10.0.16299
17 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
18 verbose node v8.9.4
19 verbose npm  v5.6.0
20 error code ELIFECYCLE
21 error errno 2
22 error vue-devtools@3.1.7 build: `cd shells/chrome && cross-env NODE_ENV=production webpack --progress --hide-modules`
22 error Exit status 2
23 error Failed at the vue-devtools@3.1.7 build script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 2, true ]`

**Comment by yoyo837** [comment]
Module parse failed: Unexpected token (32:6)

```javascript
var xxx = {
  leftStyles: function leftStyles () {
    return ( obj = {}
    var obj;, obj[this.view === 'vertical' ? 'width' : 'height'] = ((this.boundSplit) + "%"), obj )
  }
}
```

**Comment by crcatala** [comment]
Also observed this issue the other day, but looks like it was just fixed in b3f217d647e4fb98a17054ab49364ede73d60f56 by specifying the buble target option. Was previously `null` target in production:

https://github.com/vuejs/vue-devtools/blob/b3f217d647e4fb98a17054ab49364ede73d60f56/shells/chrome/webpack.config.js#L5-L8

**Comment by yyx990803** [comment]
Latest master should build without errors.
